scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/bsp/config/ScalaPluginConstants.scala

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.plugins.scala.bsp.config
+
+import com.intellij.openapi.externalSystem.model.ProjectSystemId
+import org.jetbrains.plugins.bsp.extension.points.BuildToolId
+
+object ScalaPluginConstants {
+  val ID = "scalabsp"
+  val BUILD_TOOL_ID: BuildToolId = new BuildToolId("Sbt")
```

Comment:
maybe use something like `scalabsp` or `sbtbsp` in place of `Sbt` to be more specific.

Reply:
we'll have to do some research what happens if you use a BSP server other than sbt